CMakeLists.txt

```diff
@@ -358,8 +363,23 @@ else()
   list(PREPEND VIAMCPPSDK_GRPCXX_REFLECTION_LIBRARIES ${VIAMCPPSDK_GRPCXX_REFLECTION_LIB})
 endif()
 
-find_package(xtl ${VIAMCPPSDK_XTL_VERSION_MINIMUM} CONFIG REQUIRED)
-find_package(xtensor ${VIAMCPPSDK_XTENSOR_VERSION_MINIMUM} CONFIG REQUIRED)
+include(FetchContent)
+
+FetchContent_Declare(
+  xtl
+  GIT_REPOSITORY https://github.com/xtensor-stack/xtl.git
+  GIT_TAG        e0f00666d90086bb245ae73abb6123d0e2c1b30b # 0.7.2
```

Comment:
The docs for FetchContent_Declare recommend against using tags because they aren't immutable.